controller/controller_server/utils.py

```diff
@@ -201,6 +236,34 @@ def generate_csi_create_snapshot_response(new_snapshot, source_volume_id):
     return res
 
 
+def _get_supported_capabilities(volume_capabilities):
```

Comment:
I'm confused. what is the type of `volume_capabilities`? is it really something "plural"?

Reply:
we have mount , block and access mode, all are capabilities. the type is csi_pb2 object

Reply:
so you agree that the parameter type is some Object (singular) - please change it to a singular name.
same goes for this function name, since it returns a singular object.

Reply:
done

Reply:
same for the variable of return value in caller function...

Reply:
done